test/acceptance/features/steps/steps.py

```diff
@@ -218,6 +218,21 @@ def then_app_is_connected_to_db(context, db_name):
     assert app_db_name == db_name, f"Unexpected response from API ('{db_endpoint}'): '{app_db_name}'. Expected is '{db_name}'"
 
 
+@when(u'Service binding "{sb_name}" is deleted')
+def service_binding_is_deleted(context, sb_name):
+    openshift = Openshift()
+    result = openshift.delete_service_binding(sb_name)
+    assert result is not None, f"CR {sb_name} deleted"
+
+
+@then(u'Secret "{secret_name}" does not exist')
+def secret_does_not_exist(context, secret_name):
+    openshift = Openshift()
+    time.sleep(60)
```

Comment:
instead of waiting in absolute number of second, please apply `polling2` library.

Reply:
This function removed as Secret verification is not required (there was another review comment)